packages/pwa-kit-react-sdk/bin/upload.js

```diff
@@ -62,10 +62,12 @@ try {
     Utils.fail(message)
 }
 
+const config = Utils.getConfig(argv.t)
```

Comment:
`getConfig` currently doesn't accept any arguments. It looks like we are attempting to get the config object for a given target, that will need to be fixed.